pepper-apis/src/main/java/org/broadinstitute/ddp/json/CreateTemporaryUserPayload.java

```diff
@@ -13,16 +13,29 @@
     @SerializedName("auth0Domain")
     private String auth0Domain;
 
+    @SerializedName("languageCode")
+    private String languageCode;
+
     public CreateTemporaryUserPayload(String auth0ClientId, String auth0Domain) {
         this.auth0ClientId = auth0ClientId;
         this.auth0Domain = auth0Domain;
     }
 
+    public CreateTemporaryUserPayload(@NotBlank String auth0ClientId, String auth0Domain, String languageCode) {
```

Comment:
I think you can remove the parameter annotation, don't think it does much for us.
```suggestion
    public CreateTemporaryUserPayload(String auth0ClientId, String auth0Domain, String languageCode) {
```

Reply:
Sorry, I meant you can remove annotation from constructor. Let's keep the one on the member field on L8...